platform/platform-migratable/src/main/java/org/codice/ddf/platform/migratable/impl/PlatformMigratable.java

```diff
@@ -37,7 +48,10 @@
    *
    * <p>1.0 - initial version
    */
-  private static final String CURRENT_VERSION = "1.0";
+  private static final String CURRENT_VERSION = "2.0";
```

Comment:
❗️ Can we document above what has changed in version 2.0?

Reply:
Reverted this change so that the migratable versions are remaining the same.